packages/react-native-sdk/src/providers/embedded/auth.ts

```diff
@@ -82,7 +82,7 @@ export class ExpoAuthProvider implements AuthProvider {
         const url = new URL(result.url);
         const walletId = url.searchParams.get("wallet_id");
         const organizationId = url.searchParams.get("organization_id");
-        const provider = url.searchParams.get("provider");
+        const provider = url.searchParams.get("provider") as EmbeddedProviderAuthType;
```

Comment:
This is not safe as the search parameter can also be `undefined`. We could use the earlier proposed helper to make this type safe.